### PR TITLE
Open model on startup

### PR DESCRIPTION
**Pull request by joswarmer**
Possibility to open a model using the freon URL:  _?model=modelname_

### CODE REVIEW

**Comment by dslmeinte** [comment]
Looks good, but I haven't tried it out yet, because of lack of time.

One question: how about specifying the model unit as well?

File: packages/playground/src/webapp/FreonLayout.svelte [hunk 1]
```diff
@@ -98,12 +99,27 @@
 				$modelNames = names;
 			}
 		});
-
-		if (!$userMessageOpen) {
-			// open the app with the open/new model dialog
-			$openModelDialogVisible = true;
+		// If a model is given as parameter, open this model
+		//A new modelis created when this model does not exist
```
Comment (annekekleppe) [inline packages/playground/src/webapp/FreonLayout.svelte:103]: Please clean up comment

Comment (annekekleppe) [reply packages/playground/src/webapp/FreonLayout.svelte:103]: I mean: add some spaces.

File: packages/playground/src/webapp/FreonLayout.svelte [hunk 2]
```diff
@@ -39,8 +39,8 @@
```
Comment (annekekleppe) [inline packages/playground/src/webapp/FreonLayout.svelte:39]: Looks good, but can you please add something to the documentation, both website and readme files: a short remark how to use this or simply a todo specifying that this feature needs to be documented. Nothing fancy, just a simple reminder in the right places.

**Review by annekekleppe — APPROVED** [review]

File: packages/playground/src/webapp/FreonLayout.svelte (hunk 2 above)
Comment (joswarmer) [reply packages/playground/src/webapp/FreonLayout.svelte:39]: Done

**Comment by joswarmer** [comment]
> Looks good, but I haven't tried it out yet, because of lack of time.
> 
> One question: how about specifying the model unit as well?

Merging this PR first. ModelUnit will come later.

**Pull request merged**